components/menu/themes/spectrum.css

```diff
@@ -17,7 +17,7 @@
 
 @container style(--system: legacy) {
 	.spectrum-Menu {
-		--spectrum-menu-item-background-color-default: var(--spectrum-gray-50);
+		--spectrum-menu-item-background-color-default: var(--spectrum-gray-100);
```

Comment:
With this change, what should I expect to see with the hovers? I don't visually see a difference anymore, but maybe I'm not supposed to yet.